### PR TITLE
LLVM working for powerpc replay

### PR DESCRIPTION
**Pull request by wangray**

### CODE REVIEW

**Comment by phulin** [comment]
this looks good but there are some indentation issues. can you make sure everything's spaces?

File: panda/llvm/tcg-llvm.cpp [hunk 1]
```diff
@@ -531,6 +531,9 @@ void TCGLLVMContextPrivate::setValue(int idx, Value *v)
     delValue(idx);
     m_values[idx] = v;
 
+	//Checks that bitwidth of Value equals bitwidth of pointer we're storing to
+	assert(v->getType() != cast<PointerType>(getPtrForValue(idx)->getType())->getElementType());
```
Comment (phulin) [inline panda/llvm/tcg-llvm.cpp]: this is still tabs.

Comment (wangray) [inline panda/llvm/tcg-llvm.cpp]: I think i fixed but it looks the same?

**Comment by phulin** [comment]
feel free to merge

**Comment by wangray** [comment]
Will merge after CI passes

**Comment by wangray** [comment]
Canceling queued builds so last one gets through faster

**Pull request merged**